content/docs/getting-started/connect-dbaas.md

```diff
@@ -64,3 +65,25 @@ Configure `cURL` with DBaaS credentials.
 ```sh
 alias curl="curl -s -k -u \"${NUODB_CP_USER}:${NUODB_CP_PASSWORD}\""
 ```
+
+### Setting up the Terraform Provider
+
+To use terraform to manage your databases, you will want to use the [`nuodbaas` provider](https://registry.terraform.io/providers/nuodb/nuodbaas).
+The [provider documentation](https://registry.terraform.io/providers/nuodb/nuodbaas/latest/docs#schema) covers all of the attributes you can define but, if they are not set, the provider can infer a configuration using the environment variables set above.

```

Comment:
Can you replace the " but," with a period? It flows better to split it into two separate sentences.

Reply:
It is a bit of a run on. I will split it.